recipes/bmi_dbseabed/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "bmi_dbseabed" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/gantian127/{{ name }}/archive/v{{ version }}.tar.gz
```

Comment:
Any reason to not use the PyPI sdist? It is not a problem to use GH but, when a PyPI sdist is available, we tend to prefer it. It makes a few of the automation we have in place a bit easier, like reading the package metadata and updating for newer versions.

Reply:
Sure. You are missing some nice automated services though.